Add range-v3 0.11.0 release

krnowak opened the pull request.

krnowak committed:
Add range-v3 0.11.0 release

eli-schwartz commented on upstream.wrap:
```diff
@@ -0,0 +1,5 @@
+directory = range-v3-0.11.0
+source_url = https://github.com/ericniebler/range-v3/archive/0.11.0.zip
+source_filename = range-v3-0.11.0.zip
+source_hash = 269e6a5ec8ac9967d5a052e6e312caff5e9f739b149feb402d13719af967b2d6
```
Needs https://mesonbuild.com/Wrap-dependency-system-manual.html#provide-section

Note: upstream does not provide pkg-config files, but they do provide range-v3-config.cmake, confirming the name "range-v3" for the provided dependency.

krnowak replied:
Done.

krnowak commented:
Updated. I also added the call to `meson.override_dependency` in `meson.build`. Not sure if I should do it, but seemed to make sense. This bumped the meson requirement of range-v3 to >=0.54.0.
